src/index.html

```diff
@@ -6,8 +6,285 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>John Roberts' CV</title>
     <link rel="stylesheet" href="./css/index.scss" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+    />
   </head>
+
   <body>
+    <!-- Header Section-->
+    <div class="header">
+      <a href="./index.html">Home</a>
+      <a href="">DOWNLOAD MY FULL RESUME</a>
+    </div>
+
+    <!-- Head Section-->
+    <div class="head">
+      <h1>John Roberts</h1>
+      <h2>BUSINESS EXPERT</h2>
+      <img src="" alt="" />
```

Comment:
The `alt` should describe the content of the image. So for this case, it's better to be the author name.

```
<img src="" alt="John Roberts" />
```